jobs/mesoscale/stats/JEVS_MESOSCALE_STATS

```diff
@@ -1,78 +1,126 @@
-export COMINprepbufr=${COMINprepbufr:-$(compath.py  obsproc/${obsproc_ver})}
-export COMINccpa=${COMINccpa:-$(compath.py  ccpa/${ccpa_ver})}
-export COMINsrefmean=${COMINsrefmean:-$(compath.py  $NET/$evs_ver)}/prep/$COMPONENT
-
-export COMOUT=${COMOUT:-$(compath.py -o ${NET}/${evs_ver})}/$STEP/$COMPONENT
-mkdir -p $COMOUT
-
-
-export COMOUTfinal=${COMOUT}/${MODELNAME}.${VDATE}
-export COMOUTsmall=${COMOUT}/${RUN}.${VDATE}/${MODELNAME}/${VERIF_CASE}
-mkdir -p $COMOUTfinal $COMOUTsmall
-
-
 export HOMEevs=${HOMEevs:-${PACKAGEROOT}/evs.${evs_ver}}
 export EXECevs=${EXECevs:-$HOMEevs/exec}
 export PARMevs=${PARMevs:-$HOMEevs/parm}
 export USHevs=${USHevs:-$HOMEevs/ush}
 export FIXevs=${FIXevs:-/lfs/h1/ops/prod/packages/evs.${evs_ver}/fix}
 export MASKS=$FIXevs/masks
 
-export pid=$$
-
-
-# ###################################
-# SET SHELL PROCESSING VARIABLES
-# ###################################
-export PS4='$SECONDS + '
-date
-
-
-####################################
-# Determine Job Output Name on System
-####################################
-export pgmout="OUTPUT.${pid}"
+if [ $MODELNAME = nam ] || [ $MODELNAME = rap ]; then
+# in production the following will be set using compath.py
+  export COMIN=${COMIN:-$(compath.py -o $NET/$evs_ver)}
+  export COMINnam=${COMINnam:-$(compath.py ${envir}/com/nam/${nam_ver})}
+  export COMINrap=${COMINrap:-$(compath.py ${envir}/com/rap/${rap_ver})}
+  export COMINobsproc=${COMINobsproc:-$(compath.py ${envir}/com/obsproc/${obsproc_ver})}
+  export COMINobs=$COMINobsproc
+
+# in production, the following will be set to the approp. COMOUT dir
+  export COMOUT=${COMOUT:-$(compath.py -o ${NET}/${evs_ver}/${STEP}/${COMPONENT}/${RUN})}
+  export COMOUTfinal=$COMOUT/${MODELNAME}.${VDATE}
+  export COMOUTsmall=${COMOUTsmall:-$(compath.py ${NET}/${evs_ver}/${STEP}/${COMPONENT}/${RUN}.${VDATE}/${MODELNAME}/${VERIF_CASE})}
+
+  source $config
+elif [ $MODELNAME = sref ]; then
+  export cycle=t${cyc}z
+
+  setpdy.sh 
+  . $DATA/PDY
+  export VDATE=${VDATE:-$PDYm2}
 
+  export COMIN=${COMIN:-$(compath.py  $NET/$evs_ver)}/prep/$COMPONENT/$RUN
+  export COMINsref=${COMINsref:-$(compath.py  sref/$sref_ver)}
+  export COMINprepbufr=${COMINprepbufr:-$(compath.py  obsproc/${obsproc_ver})}
+  export COMINccpa=${COMINccpa:-$(compath.py  ccpa/${ccpa_ver})}
+  export COMINsrefmean=${COMINsrefmean:-$(compath.py  $NET/$evs_ver)}/prep/$COMPONENT
+
+  export COMOUTfinal=${COMOUT}/${MODELNAME}.${VDATE}
+  export COMOUTsmall=${COMOUT}/${RUN}.${VDATE}/${MODELNAME}/${VERIF_CASE}
+  mkdir -p $COMOUTfinal $COMOUTsmall
+else
+  echo "Improper MODELNAME: $MODELNAME"
+  exit
+fi
 
-# STEP->prep/stats, MODELNAME->sref, COMPONENT->mesoscale, VERIF_CASE->grid2grid/grid2obs/precip
-$HOMEevs/scripts/${COMPONENT}/${STEP}/exevs_${MODELNAME}_${VERIF_CASE}_${STEP}.sh
 
+# Execute the Script
+$SCRIPTSevs/$COMPONENT/$STEP/exevs_${MODELNAME}_${VERIF_CASE}_${STEP}.sh
+export err=$?; err_chk
+
+
+
+#  if [ $MODELNAME = nam ] || [ $MODELNAME = rap ]; then
```

Comment:
Go ahead and remove the next section of commented lines now that this processing (copying of small stat files to $COMOUTsmall) is happening correctly downstream in the scripts.